submissions/lalalilulu/hooli-style_popup/index.html

```diff
@@ -0,0 +1,108 @@
+                            <span>Videos</span>
+                        </div>
+                        <div class="popup-item">
+                            <img class="popup-item-icon" src="icons/search.png" alt="apps">
+                            <span>Search</span>
+                        </div>
+                        <div class="popup-item">
+                            <img class="popup-item-icon" src="icons/photos.png" alt="apps">
+                            <span>Photo</span>
+                        </div>
+                        <div class="popup-item">
+                            <img class="popup-item-icon" src="icons/apps.png" alt="apps">
+                            <span>Apps</span>
+                        </div>
+                        <div class="popup-item">
+                            <img class="popup-item-icon" src="icons/documents.png" alt="apps">
+                            <span>Document</span>
+                        </div>
+                    </div>
+                <div class="more-open">
+                    <div class="more-text">More</div>
+                    <div class="more">
+                        <div class="more-item">
+                            <img class="more-item-icon" src="icons/chat.png" alt="photos">
+                            <span>Chat</span>
+                        </div>
+                        <div class="more-item">
+                            <img class="more-item-icon" src="icons/classroom.png" alt="photos">
+                            <span>Classroom</span>
+                        </div>
+                        <div class="more-item">
+                            <img class="more-item-icon" src="icons/collection.png" alt="photos">
+                            <span>Collection</span>
+                        </div>
+                        <div class="more-item">
+                            <img class="more-item-icon" src="icons/mars.png" alt="photos">
+                            <span>Mars</span>
+                        </div>
+                        <div class="more-item">
+                            <img class="more-item-icon" src="icons/blogs.png" alt="photos">
+                            <span>Blogs</span>
+                        </div>
+                        <div class="more-item">
+                            <img class="more-item-icon" src="icons/promo.png" alt="photos">
+                            <span>Promos</span>
+                        </div>
+                        <div class="more-last-item">
+                            <a href="https://kottans.org/" class="kottans">Even more from Kottans</a>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+            <img class="icon" src="icons/bell.png">
+            <img  class="avatar" width="24" height="24" src="icons/avatar.png">
+    </nav>
+</header>
+</body>
+</html>
```

Comment:
Add an empty line to the end of each file. [Reason](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline)